### PR TITLE
111 UI when the user hasnt uploaded a cv yet the user page is not centered

### PR DESCRIPTION
**Pull request by Aradon267**
image

### CODE REVIEW

File: cv_next/app/auth/callback/route.ts [hunk 1]
```diff
@@ -16,10 +16,13 @@ export async function GET(request: Request) {
...
         return NextResponse.redirect(
-          `${origin}${Definitions.AUTH_DEFAULT_REDIRECT}${next}`
+          `${process.env.NEXT_PUBLIC_BASE_URL}${Definitions.AUTH_DEFAULT_REDIRECT}${next}`
         );
       } else {
-        const notFoundUrl = new URL("/not-found", origin);
+        const notFoundUrl = new URL(
+          "/not-found",
+          process.env.NEXT_PUBLIC_BASE_URL
+        );
```
Comment (YechiamW3) [inline cv_next/app/auth/callback/route.ts:25]: Is this change related to this PR?
Also - I recall a similar change in a different PR, please check.

Comment (Aradon267) [reply cv_next/app/auth/callback/route.ts:25]: Isn't related, I'll check where it was added because I think that it was

**Review by YechiamW3 — APPROVED** [review]

**Review by TheGreenOak — APPROVED** [review]
